src/goose.rs

```diff
@@ -1434,44 +1477,45 @@ impl GooseUser {
     ///     let mut task = task!(loadtest_index_page);
     ///
     ///     async fn loadtest_index_page(user: &GooseUser) {
-    ///         let mut goose = user.get_named("/", "index").await;
-    ///         // Extract the response Result.
-    ///         match goose.response {
-    ///             Ok(r) => {
-    ///                 // Grab a copy of the headers so we can include them when logging errors.
-    ///                 let headers = &r.headers().clone();
-    ///                 // We only need to check pages that returned a success status code.
-    ///                 if !r.status().is_success() {
-    ///                     match r.text().await {
-    ///                         Ok(html) => {
-    ///                             // Server returned an error code, log everything.
-    ///                             user.log_debug(
-    ///                                 "error loading /",
-    ///                                 Some(goose.request),
-    ///                                 Some(headers),
-    ///                                 Some(html.clone()),
-    ///                             );
-    ///                         },
-    ///                         Err(e) => {
-    ///                             // No body was returned, log everything else.
-    ///                             user.log_debug(
-    ///                                 "error loading /",
-    ///                                 Some(goose.request),
-    ///                                 Some(headers),
-    ///                                 None,
-    ///                             );
+    ///         if let Ok(mut goose) = user.get_named("/", "index").await {
```

Comment:
Maybe - adjust those patterns as well to the new pattern and not the HUGE indentation change ;)

Reply:
It seems my grepping skills betrayed me horribly, thanks

Reply:
fixed